android/anc/src/main/java/org/smartregister/fhircore/anc/ui/anccare/details/AncDetailsViewModel.kt

```diff
@@ -33,6 +35,11 @@ class AncDetailsViewModel(
   val patientId: String
 ) : ViewModel() {
 
+  val LIBRARY_URL = "https://fhir.labs.smartregister.org/fhir/Library?_id=1752"
```

Comment:
can we move these to a config file and out of the code?

Reply:
@pld This is done